.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: android
+sudo: true # travis-ci/travis-ci#5582 implies this could help against killed build-jobs
```

Comment:
travis-ci/travis-ci#5582 is not really conclusive, but we can try this. However, if builds still fail, let's revert this.